src/Components/ArticleDetails.jsx

```diff
@@ -0,0 +1,67 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+
+function ArticleDetails() {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState([]);
+  const [comments, setComments] = useState([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
+      .get(`https://news-project-fe7s.onrender.com/api/articles/${article_id}`)
+      .then((response) => {
+        setArticle(response.data.article);
+        setIsLoading(false);
+        setIsError(false);
+      })
+      .catch((err) => {
+        setIsError(true);
+        setIsLoading(false);
+      });
+
+    //fetching comments from articles
+    axios
+      .get(
+        `https://news-project-fe7s.onrender.com/api/articles/${article_id}/comments`
+      )
+      .then((response) => {
+        setComments(response.data.comments);
+      })
+      .catch((err) => {
+        setIsError(true);
+      });
+  }, [article_id]);
+
```

Comment:
Could you abstract your axios requests to another file ? i.e. api.js in a util folder or something ? It would allow you to reuse these functions elsewhere as well as be more readable.